src/DataLoaders.py

```diff
@@ -0,0 +1,102 @@
+# Classes for wrapping data into datasets
+
+import pandas as pd
+import numpy as np
+import biotite
+import biotite.structure as struc
+import src.DataProcessing  as dp
+from tqdm import tqdm
+from typing import Optional, Literal
+from torch_geometric.data import Dataset
+
+class ProteinDataset(Dataset):
+    """
+    Dataset for loading proteins from the paper given splits.
+    """
+    def __init__(self, labels, database, include, flex: Literal["msqf", "bfact", "pseudo"] = "msqf", exclude: Optional[str] = None):
+        self.flex = flex
+        self.database = database
+        self.structures = []
+        self.protein_labels = pd.DataFrame(columns = ['chain_id', 'label'])
+        if exclude != None:
+            structure_set = set(line.strip() for line in open(include)).difference(set(line.strip().replace("_",".") for line in open(exclude)))
```

Comment:
Please break lines like these into multiple lines with simpler statements & temporary variables that are easy to read. In the end readability is much more important than a minute gain in efficiency. For general engineering hints  I highly recommend you to take a look at [Google's python guidelines](https://google.github.io/styleguide/pyguide.html#27-comprehensions--generator-expressions).

Also, as FYI, it is best practice to always use the `with` context manager in python when opening files (c.f. [here](https://stackoverflow.com/questions/18840880/will-using-list-comprehension-to-read-a-file-automagically-call-close) for more info) and [here](https://google.github.io/styleguide/pyguide.html#311-files-sockets-and-similar-stateful-resources).